### ISSUE TITLE
fragmented slide?

### ISSUE THREAD

**Issue by c0b** [webslides/WebSlides#92]
as I am looking for reveal.js alternatives to craft beautiful presentation, this project can be counted as one, but still lacks some important feature like #91 ; and may also include the fragmented slide?

like http://lab.hakim.se/reveal-js/#/fragments want the slide not all showing at once, but fragments to step through one by one

**Comment by c0b** [comment]
this is an important and elementary feature, if you consider all the competing, js and web based presentation framework projects at https://bestof.js.org/tags/presentation , most others already have; 

also in the traditional desktop based microsoft powerpoint, or the Open Office, or in google office, none of them not having this feature

**Comment by Belelros** [comment]
Let me try to say this politely. We couldn't care less of what others already have. Really. Trying to ask for a feature just because others have it even when we already said we were not going to makes little sense.

If you need that feature desperately you have a two options:

1. Use one of the *competing* presentation frameworks. I'm not sure about them but I don't think we're here to compete. We just offer different things and visions. We won't be offended if you stop using WebSlides or not use it and chose Reveal instead. It just means WebSlides isn't for you.
2. Fork WebSlides, then implement your feature, then rename it and finally release it. 

We have already said we won't go into that direction which is what we'll do: not do it. We're sorry if you stop considering WebSlides as an "alternative to craft beautiful presentations".

**Comment by c0b** [comment]
don't get me wrong, I noticed from [`contributor graph`](https://github.com/webslides/WebSlides/graphs/contributors) that this project is a new project started in this year 2017, get 3K stars in just 8 months is an amazing achievement;
all issues reporters I believe are in the good belief for this project can grow into a better one, and from github open source experience, whoever reported the issue is also the very potential one can help implement; for me also did a lot of open source contribution, but ask the question first for a test if anyone is doing this feature and in progress

![image](https://user-images.githubusercontent.com/14798161/29284394-6bee65ba-80df-11e7-92c5-6ea3906c7c35.png)

however once somebody suggested to bestof.js and people found it from https://bestof.js.org/tags/presentation or http://alternativeto.net/software/reveal-js/?license=opensource this is inevitably seen as alternative solutions to each other, or competing projects, whatever you think you are competing or not; people are thinking in that way.
it also depends on what do you think future of this project, just for fun without profit? or based on donations to cover some spent time, or to commercialize in future is one option? then will compete with revealjs authors' https://slides.com/about anyway

from https://medium.freecodecamp.org/introducing-webslides-fa7a9e37ff97 I read this, so far the pros of webslides is **landings**, **portofolios**, and **long-forms**
I hope in other cons areas webslides can catch up soon

![image](https://user-images.githubusercontent.com/14798161/29284780-df054a54-80e0-11e7-8f1d-6de8769e754c.png)

**Comment by Belelros** [comment]
Well, don't get me wrong but the _amazing_ achievement was achieved with a basic set of features and we have improved ever since. We've received positive feedback and the amount of requests like this one is fairly low.

We might, of course, be wrong with our vision but we want to learn about that and follow this path we decided together. So far, contribution from outside of the project members has been fairly minimal and that's ok. 

We are not aiming to the Reveal target which is why we don't want to copy any of the Reveal features. I do believe Reveal does a really good job on some areas and I also believe we do a really good job on some areas as well, not only in those you mention.

Future will tell. We're having fun and that's all that matters for us. Copying Reveal is not fun or entertaining for us so implementing some feature just because X has it *won't ever happen*. 

We do consider WebSlides to be mostly done to the point we wanted to take it. What will happen from here, future will tell.
